File: app/src/main/java/eu/napcode/gonoteit/model/UserModel.java
package eu.napcode.gonoteit.model;

public class UserModel {

    private String userName;

    public UserModel(String userName) {
        this.userName = userName;
    }

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }
}

File: app/src/main/java/eu/napcode/gonoteit/model/UserModel.kt
package eu.napcode.gonoteit.model

import eu.napcode.gonoteit.dao.user.UserEntity
import eu.napcode.gonoteit.data.user.favoritesMapToList

class UserModel(var userName: String) {
    var favorites: List<Long> = mutableListOf()

    constructor(userEntity: UserEntity) : this(userEntity.name) {
        favorites = favoritesMapToList(userEntity.favorites)
    }
}
